cluster/gce/gci/configure-helper.sh

```diff
@@ -506,7 +517,22 @@ function start-kubelet {
   if [[ -n "${FEATURE_GATES:-}" ]]; then
      flags+=" --feature-gates=${FEATURE_GATES}"
   fi
-  echo "KUBELET_OPTS=\"${flags}\"" > /etc/default/kubelet
+
+  # Write the systemd service file for kubelet.
+  cat <<EOF >/etc/systemd/system/kubelet.service
+[Unit]
+Description=Kubernetes kubelet
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+Restart=always
+RestartSec=10
+ExecStart=${kubelet_bin} ${flags}
```

Comment:
Can we continue use the /etc/default/kubelet for flags? What we currently have is:

```
[Service]
Restart=always
RestartSec=10
EnvironmentFile=/etc/default/kubelet
ExecStartPre=/bin/mkdir -p /etc/kubernetes/manifests
ExecStart=${kubelet_bin} $KUBELET_OPTS
```

Additionally, if you want, you can even do something like:
```
ExecStart=$KUBELET_BIN $KUBELET_OPTS
```
and set KUBELET_BIN also in /etc/default/kubelet. This can allow you to keep the kubelet.service file completely static.